UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/About.cshtml

```diff
@@ -0,0 +1,120 @@
+﻿@{
+    Layout = "~/Views/Shared/_MSILayout.cshtml";
+}
+@{
+    ViewData["Header"] = "About";
+}
+<h4>
+    ADMIRALTY Notices to Mariners (NMs)
+</h4>
+<p>
+    ADMIRALTY charts and publications should be maintained so that they are fully up-to-date for the latest safety-critical navigational information. The ADMIRALTY Notices to Mariners service provides all the data you need to maintain our products.
+</p>
+<p>
+    ADMIRALTY NMs contain all the corrections, alterations and amendments for the UKHO's worldwide series of ADMIRALTY charts and publications. They are published weekly in several formats but best known as the weekly booklets, which are dispatched directly from the UKHO.
```

Comment:
Not sure if we correct it or keep it same as existing?

```suggestion
    ADMIRALTY NMs contain all the corrections, alterations and amendments for the UKHO's worldwide series of ADMIRALTY charts and publications. They are published weekly in several formats but best known as the weekly booklets, which are despatched directly from the UKHO.
```

Reply:
I just checked...`dispatch` is the American spelling and `despatch` is the British one so I've just leaned something :)